docs/api/structures/user-agent-metadata.md

```diff
@@ -0,0 +1,12 @@
+# UserAgentMetadata Object
+
+* `brands` [UserAgentBrandVersion[]](user-agent-brand-version.md) - The brand and versions.
+* `fullVersionList` [UserAgentBrandVersion[]](user-agent-brand-version.md) - The brand and full versions.
+* `fullVersion` string - full browser version.
+* `platform` string - The platform.
+* `platformVersion` string - The platform version.
+* `model` string - The model of mobile device.
+* `mobile` boolean - Whether the Electron is running on a mobile device.
```

Comment:
I don't think we should expose these, since they aren't applicable to Electron, and may cause confusion.

Reply:
These properties are useful when emulating mobile devices, replacing the Electron with the user agent should make it less confusing.